css/style.css

```
@@ -1,9 +1,217 @@
-
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+/* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
 
 body {
-    font-family: 'Roboto', sans-serif;
-    -webkit-font-smoothing: antialiased;
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+  font-size: 16px;
+  
+    
+
+}
+/* Header Section */
+header {
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+}
+header img {
+  height: 2.4rem;
+  padding: 1rem 0 1rem 2rem;
+}
+
+/* Navigation Section */
+
+nav ul {
+  display: flex;
+  justify-content: space-evenly;
+  list-style: none;
+  width: 34rem;
+}
+
+nav ul a {
+  text-decoration: none;
+  color: #9c9fa7;
+}
+
+nav ul a:hover {
+  color: #e0633a;
+}
+li:first-child a {
+  color: rgb(117, 114, 114);
+  font-weight: 500;
+}
+
+/* Main Content Section */
+.hero {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-image: url("../img/first-background.jpg");
+  width: 73rem;
```

```suggestion
  max-width: 73rem;
```
A set width of `73rem` means when we shrink the screen size then the width stays at exactly `73rem`. When the screen is 600px wide for example, a horizontal scrollbar to appears because the hero section too wide for the screen.

Changing this to `max-width` means that the width can decrease below `73rem` which means on less wide screens the `Introducing Karma` text will still be centred.

I remember you asked about why I used `max-width` instead of `width` in the CSS Wikipedia devtools exercise, and this is the reason why! I just forgot about it then